socorrolib/__init__.py

```diff
@@ -1,4 +1,4 @@
-from pkg_resources import get_distribution
+from pkg_resources import get_distribution, DistributionNotFound
```

Comment:
This fixes the exception you get because `DistributionNotFound` isn't in the name space in the case when an exception is thrown from the `get_distribution` call.